### PR TITLE
Doing this in a separate PR

### PR DESCRIPTION
**Pull request by blakeparkinson**

### CODE REVIEW

File: models/players.js [hunk 1]
```diff
@@ -222,6 +222,17 @@ function mongoInsertPlayers(team_id, team_document){
   });
 }
 
+function mongoBulkInsertPlayers(document){
+    db.open(function(err, db){
+      db.collection("players").insert(document, function(err){
```
Comment (kelseyedmodo) [inline models/players.js]: Why is this an insert not an upsert? Do we really only want to do this once, or set it up to update info each night/week etc.?

File: scripts/players_import.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+options.only_insert = true;
+var rosters = [];
+
+    //fetch the team collections
+    db.collection('teams').find({'league': league}).toArray(function (err, teams) {
+      //fetch the players collectiom
+      db.collection('players').find().toArray(function (err, players){
+        var common_team_ids = [];
+        var all_team_ids = [];
+        //loop through and filter out the players collections that we already have
```
Comment (kelseyedmodo) [inline scripts/players_import.js]: But player info changes. Don't we want to check for last updated dates and filter on that, instead of simple existence?

**Comment by kelseyedmodo** [comment]
I can't run this branch right now. Get the following error when I npm start (I've git pull -ed   so everything is uptodate):

> sports@0.0.4 start /Users/kelsey/projects/sports2/sports2
> node ./bin/www

dyld: lazy symbol binding failed: Symbol not found: _node_module_register
  Referenced from: /Users/kelsey/projects/sports2/sports2/node_modules/mongojs/node_modules/mongodb/node_modules/bson/build/Release/bson.node
  Expected in: dynamic lookup

dyld: Symbol not found: _node_module_register
  Referenced from: /Users/kelsey/projects/sports2/sports2/node_modules/mongojs/node_modules/mongodb/node_modules/bson/build/Release/bson.node
  Expected in: dynamic lookup

Trace/BPT trap: 5

**Comment by blakeparkinson** [comment]
@kelseyedmodo murge conflicts resolved, and it works well with your new encryption stuff!

**Comment by kelseyedmodo** [comment]
:icecream:  rate limited so I could only get a nice bulk upload of 2 teams... but they looked good!

**Pull request merged**